source/runner/Executor.ts

```diff
@@ -185,3 +190,28 @@ export class Executor {
     }
   }
 }
+
+const compliment = () => {
+  const compliments = ["Well done.", "Congrats.", "Woo!", "Yay.", "Jolly good show.", "Good on 'ya.", "Nice work."]
+  return compliments[Math.floor(Math.random() * compliments.length)]
+}
+
+const pluralize = (word: string, array: any[]) => `${array.length} ${word}${array.length ? "s" : ""}.`
+
+const messageForResults = (results: DangerResults) => {
+  if (results.fails.length && results.warnings.length) {
+    return `All green. ${compliment()}`
+  }
+
+  let message = "⚠️ "
+  if (results.fails.length) {
+    message += pluralize("Errors", results.fails)
```

Comment:
Possible bug (if I'm reading this function correctly): when there are 0 errors, `pluralize()` will return "0 Errors"; when there is more than 1 error, `pluralize()` will return "n Errorss" with two **s**'s.

Looking at some example success and failure statuses I see on GitHub:

![image](https://user-images.githubusercontent.com/2344137/27044690-782cd414-4f6c-11e7-9234-3603b0e38b79.png)

![image](https://user-images.githubusercontent.com/2344137/27044708-8cb47f7c-4f6c-11e7-88b1-5813b70df5e8.png)

Thoughts on stating "⚠️  Danger found some issues. Don't worry, everything is fixable." for any warnings or errors? I feel like that wording is generic enough for all cases and simplifies this function.